### DISCUSSION TITLE
GitHub Markdown now supports LaTeX

### DISCUSSION THREAD

**Discussion by redeboer** [ComPWA/polarimetry#26] [Show and tell]
One of the large disadvantages of GitHub vs GitLab has finally been addressed 🎉 
https://github.blog/changelog/2022-05-19-render-mathematical-expressions-in-markdown

$$\Lambda_c \to p \pi K$$

**Comment by mmikhasenko** [comment]
I had wrong notification settings and missed this message 17 days ago. I learned about it just recently.
It is cool!
